### ISSUE TITLE
GPM not reachable by plugins page in admin

### ISSUE THREAD

**Issue by kmcital** [getgrav/grav#1120]
(related to issue ![1008](1008) )
My site has been fine, until I just upgraded to latest version of Grav and plugins. Then I got this exact same error.

>An exception has been thrown during the rendering of a template ("GPM not reachable. Please check your internet connection or check the Grav site is reachable") in "partials/plugins-list.html.twig" at line 19

When I issue the command `curl -I https://getgrav.org/downloads/plugins.json` from the web server it comes back with 

```
HTTP/1.1 200 OK
Server: nginx
Date: Thu, 20 Oct 2016 16:38:59 GMT
Content-Type: application/json
Connection: keep-alive
Vary: Accept-Encoding
Pragma: no-cache
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: OPTIONS, GET, HEAD, POST, PUT, DELETE
Cache-Control: max-age=3600
Expires: Thu, 20 Oct 2016 17:38:59 GMT
Vary: Accept-Encoding
Set-Cookie: grav-site-7fe36eb=2c03fb400e6d7c6e006e58f65e1e6328; expires=Thu, 20-Oct-2016 17:08:59 GMT; Max-Age=1800; path=/; domain=getgrav.org
Set-Cookie: grav-site-7fe36eb=2c03fb400e6d7c6e006e58f65e1e6328; expires=Thu, 20-Oct-2016 17:08:59 GMT; Max-Age=1800; path=/; domain=getgrav.org; HttpOnly
Last-Modified: Thu, 08 Oct 2015 22:20:13 GMT
ETag: 80fef8abc41eff6c507d08b627a0fbe4
```

Also, with `bin/gpm index` I now get "No data was found in the GPM repository stored locally.". I've followed the permissions instructions on the "Cannot connect to the GPM" page.

This happened directly as a result of the upgrades I did.

**Comment by flaviocopes** [comment]
In the System settings, try turning off "Remote Verify Peer (SSL)", does it change something? (`gpm.verify_peer: false`) in system.yaml)

**Comment by kmcital** [comment]
That setting fixes the problem. I'm back in business, thanks @flaviocopes !!

As a point of clarity, I had only upgraded Grav, not the plugins yet when the error surfaced. Now I was able to update all plugins.

**Comment by rhukster** [comment]
We're working on a fix for this.  We're explicitly setting verify peer either true or false. Default is true (like it is default curl) However, we never set this value before, so I guess there are some systems that had the default setting of false.  So we're not going to set this unless false is explicitly needed.  Next release will have this fix.

**Comment by flaviocopes** [comment]
Fixed in Grav 1.1.8
